### ISSUE TITLE
Notes and suggestions

### ISSUE THREAD

**Issue by Scdtp** [ejayjayjayjayjay/v2-portfolio-btstrap#1]
**SUGGESTIONS AND INPUTS:**
A blessed day Ejay, here are some of my suggestions and inputs with regards to your project.
1. I really like the minimalistic design. 
2. The color combination was great and green really suites the feel of the site. 
3. One of the things that I noticed was that there was a side scrolling happening on both desktop and mobile version of the site. 
4. You might need to look for an element that is causing the overflow. 
5. I would suggest replacing the lorem ipsum with actual text that you will be using for showcasing and marketing your services. 

Over all, the site features were great. Keep up the good work.

**Comment by ejayjayjayjayjay** [comment]
> 

yes sir . i just put (overflow-x: hidden;) in body , shout out to stackoverflow site haha
